app/javascript/components/Discover/Nav.tsx

```diff
@@ -10,13 +11,21 @@ export const Nav = ({
   currentTaxonomyPath,
   onClickTaxonomy,
   footer,
+  forceDomain = false,
 }: {
   wholeTaxonomy: Taxonomy[];
   currentTaxonomyPath?: string | undefined;
   onClickTaxonomy: (taxonomySlugPath?: string) => void;
   footer?: React.ReactNode;
+  forceDomain?: boolean;
 }) => {
-  const menuItems = React.useMemo(() => generateTaxonomyItemsForMenu(wholeTaxonomy), [wholeTaxonomy]);
+  const { discoverDomain } = useDomains();
+  const discoverUrl = Routes.discover_url({ host: discoverDomain });
+
+  const menuItems = React.useMemo(
+    () => generateTaxonomyItemsForMenu(wholeTaxonomy, forceDomain ? discoverUrl : ""),
+    [wholeTaxonomy, discoverUrl],
+  );
   const selectedCategory = menuItems.find((menuItem) => menuItem.href === `/${currentTaxonomyPath ?? ""}`)?.key;
```

Comment:
displaying the selected category would break when `forceDomain` is `true`, right?

Reply:
It's different thing actually, I need to remove the trailing slash

Reply:
I push a fix, can you check again? @ershad